src/main/scala/org/phenoscape/owlery/Main.scala

```diff
@@ -80,8 +80,13 @@ object Main extends App with SimpleRoutingApp with CORSDirectives {
 
   val conf = ConfigFactory.load()
   val serverPort = conf.getInt("owlery.port")
+  val serverHost = conf.getString("owlery.host")
 
-  startServer(interface = "localhost", port = serverPort) {
+  if(serverHost == null || serverHost.isEmpty()) {
+    val serverHostval :String = "localhost"
```

Comment:
I don't think serverHostval is being used to start the server here the way it is coded. But as I said in my other comment I think you can leave this out and we'll just require the value to be in the config.